meshtastic/interfaces/ble/management_compat_service.py

```
@@ -0,0 +1,741 @@
+            except Exception:  # noqa: BLE001 - compatibility resolution is best effort
+                logger.debug(
+                    "Error resolving _get_management_command_handler; falling back to direct handler field.",
+                    exc_info=True,
+                )
+            if callable(get_handler) and not _is_unconfigured_mock_callable(
+                get_handler
+            ):
+                try:
+                    resolved = get_handler()
+                except (
+                    Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)